### DISCUSSION TITLE
Suggest removing "Projects that use osu!framework" section of readme

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu-framework#5752] [Q&A]
There's already an integrated "Used by" section on the right side:

![firefox_5X102I1PwP](https://user-images.githubusercontent.com/35318437/232853524-edd46a01-070a-44f4-88ab-e54a6c778333.png)

You can't really sort the repos yet, so another option is just listing notable ones. The currently listed repos (excluding osu!) are not maintained and one even changed frameworks.

**Comment by peppy** [comment]
Hmm, I'd still likely to list larger projects which use osu!framework to highlight them as use cases.
